src/components/PostCard/PostCard.jsx

```diff
@@ -132,51 +136,81 @@ export const PostCard = ({ post }) => {
             )}
           </div>
           <div className="edit-post">
-            {post.username === userObj.username && <button>Edit</button>}
+            {post.username === userObj.username && (
+              <button
+                onClick={() => EdiPostClickHandler()}
```

Comment:
Suggestion: try to keep the naming of the normal javascript function in camelCase with the first letter in lowercase and components should only be in PascalCase. no hard and fast rule though

Reply:
yes. trying hard to get rid of my bad habits about this. will apply in the final cleaup round.